nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestPictureResize.java

```diff
@@ -90,4 +99,36 @@ public void testResizer() throws Exception {
 
         assertNotNull(result);
     }
+    
+    @Test
+    public void testResizerForDoc() throws Exception{
+        // TODO: NXP-27622
+        assumeFalse(SystemUtils.IS_OS_MAC);
+        
+        DocumentModel pictureDoc = session.createDocumentModel("/", "testpicture", "Picture");
+        Blob source = Blobs.createBlob(FileUtils.getResourceFileFromContext("images/test.jpg"), "image/jpeg");
+        pictureDoc.setPropertyValue("file:content", (Serializable) source);
+        pictureDoc = session.createDocument(pictureDoc);
```

Comment:
Please add a comment in the test explaining what's specific about the setup that we want to test.